Task/test online

adael2 opened the pull request:
What's this PR do?

This is a example test running the application, also give a How to test for future testing.

Where should the reviewer start?

The reviewer can start from HOW TO TEST. Then you can view the folder Test

How should this be manually tested?

Make a deploy for the application.

Any background context you want to provide?

Any comments will be review at instant

This template was adapted ~retrived~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)

Commit:
Test of Log in

This commit do a test of login launching the application.

Commit:
Add a how to test file

This commit add a how to test for the future test in the applications.

erikiado commented on express-app/routes/urls.json:
```diff
@@ -1,10 +1,10 @@
 {
 	"home": "/",
-	"apiUrl" : "http://127.0.0.1:8000",
+	"apiUrl" : "http://bloque-adal.c9users.io",
```
This should not be this URL, change it to a configuration file

erikiado commented on express-app/views/partials/sidenav.hbs:
```diff
@@ -4,5 +4,6 @@
         <li class="current"><a href="#"><i class="glyphicon glyphicon-home"></i> Dashboard</a></li>
         <li><a href="#"><i class="glyphicon glyphicon-pencil"></i> Editors</a></li>
         <li><a href="#"><i class="glyphicon glyphicon-tasks"></i> Forms</a></li>
+        <li><a href="#"><i class="glyphicon glyphicon-log-out"></i> Log Out</a></li>
```
This doesn't do nothing or does it?

erikiado commented on test/login.test.js:
```diff
@@ -0,0 +1,36 @@
+
+  beforeEach(function () {
+    this.app = new Application({
+      path: '../jp2-win32-x64/jp2.exe'
```
this should also be in the config file

erikiado commented on test/login.test.js:
```diff
@@ -0,0 +1,36 @@
+
+    //click on signin button
+    submit.click();
+  });
```
maybe put this as a test boilerplate, but this doesn't test the login

Commit:
Requested Changes

This commit resolves the request changes, I put the defult url.
I delete the log out button. I find a better form to use launch the
app without do a production. I create all possible test for log in.

Commit:
Add comments to test.

This commit is to add comments in the login test to explain what is happend.

Commit:
Delete How to do a test.

I delete this md because is better explain in the login test.
